cluster/examples/kubernetes/ceph/monitoring/monitoring-rbac.yaml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
this file doesn't need to be prefixed with "monitoring" if it is in the `monitoring/` directory. How about just `rbac.yaml`?

Reply:
Sure.